articles/actions.md

```diff
@@ -196,7 +196,7 @@ State transitions triggered by the action client:
 
 - **send_goal** - A goal is sent to the action server.
 The state machine is only started if the action server *accepts* the goal.
-- **cancel_goal** - Request that the action server stop processing the goal.
+- **request_cancel** - Request that the action server stop processing the goal.
```

Comment:
@jacobperron should the section below on [`Cancel goal service`](https://github.com/ros2/design/pull/226/files?short_path=d9d51a6#diff-0890511cd0323c94976368b725d6ada2R234) be updated too?

Reply:
Yeah, if we move forward with this renaming then that makes sense. We should probably also update the [diagram](http://design.ros2.org/articles/actions.html#middleware-implementation) above it too.